server/src/controller/PlaceController.hh

```diff
@@ -199,8 +199,8 @@ class PlaceController : public oatpp::web::server::api::ApiController
     {
         OATPP_ASSERT_HTTP(
           auth_object->user_role == Role::Admin ||
-            (dto->owner_id == auth_object->user_id &&
-             dto->owner_id == service_.getOne(id)->owner_id),
+            ((!dto->owner_id || dto->owner_id == auth_object->user_id) &&
+             auth_object->user_id == service_.getOne(id)->owner_id),
```

Comment:
`dto->owner_id` can be null as it's not required in PATCH method (we should check `user_id` from `auth_object` as it cannot be null)

Reply:
wait, I need to change it as it's wrong

Reply:
changed (why ISO C++ forbids omitting middle operand in tenary T~T)

Reply:
For me it still allows to change someone else's offer and parking place.

Reply:
Check again. It does not allow me to PATCH neither a place nor an offer which does not belong to authorized user